lib/v8debugapi.js

```diff
@@ -277,6 +280,12 @@ module.exports.create = function(logger_, config_) {
     // TODO: need test coverage for the above
     // TODO: work with debug server to come up with a more robust solution
 
+    if (breakpoint.location.line >= matchingScript.line_ends.length) {
+      return setErrorStatusAndCallback(cb, breakpoint,
+        StatusMessage.BREAKPOINT_SOURCE_LOCATION,
+        messages.INVALID_LINE_NUMBER);
```

Comment:
This patch appears to just check for scripts loaded by v8. We will have to make sure it applies generally when we move away from using v8 to find scripts.